content/guides/development.md

```diff
@@ -59,6 +60,24 @@ serve
 
 After each compilation, you will need to manually refresh your browser to see the changes.
 
+#### Watch Mode with Chrome DevTools Workspaces
```

Comment:
It might be a good idea to revise the header hierarchy in this article. `##`/`###` should be enough.